cmd/flux/check_test.go

```diff
@@ -28,13 +28,11 @@ func TestCheckPre(t *testing.T) {
 
 	cmd := cmdTestCase{
 		args:            "check --pre",
-		wantError:       false,
 		testClusterMode: ExistingClusterMode,
-		templateValues: map[string]string{
+		check: expectGoldenTemplateFile("testdata/check/check_pre.golden", map[string]string{
```

Comment:
Would be better id we use `assert` instead of `check`?

Reply:
I have renamed all uses of `check` to `assert`. Also, your comment prompted me to do the same for `expect` which I also renamed to `assert`.  My recollection is that `expect` typically proceeds to keep running the test after a failure, where `assert` stops the test in place, and I think these are more like assertions than expectations.